build.gradle

```diff
@@ -212,6 +214,37 @@ subprojects {
 
     project.tasks['sonarqube'].dependsOn codeCoverageReport
 
+
+    def pactBrokerUrl = "http://pact-broker.platform.hmcts.net "
+
+    pact {
+        serviceProviders {
+            'feeRegister_lookUp' {
```

Comment:
Why we have feeRegister_lookUp which is the api name in build.gradle?

Reply:
This is the provider name we are using to keep inline with our naming conventions (https://tools.hmcts.net/confluence/display/RSE/Naming+Conventions).  This shouldn't affect anything, this is how provider tests identifies which json to pick from pact broker.https://pact-broker.platform.hmcts.net